src/App.scss

```diff
@@ -0,0 +1 @@
+@import './Footer/Footer.scss';
```

Comment:
Let's not import this here and just import the Footer.scss file into Footer.tsx directly :)

Reply:
This has been actioned. :)